db/migrate/20180125230905_change_totp_timestamp_to_integer.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeTotpTimestampToInteger < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :users, :totp_timestamp, :timestamp
+    add_column :users, :totp_timestamp, :integer
```

Comment:
Is this going to cause downtime while the migration is being performed?

Reply:
Both versions of the db scheme are compatible with both versions of the code so it shouldn't.